### DISCUSSION TITLE
The problem occurs when I debug this repo

### DISCUSSION THREAD

**Discussion by ghost** [imgproxy/imgproxy#811] [Ideas]
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/36855501/156770826-0d324b60-0bbd-4b5b-9df7-e2355febc943.png">

This compile option will remove symbols for debugging.  
I don't think it was a mistake, since building a simple executable file by default is fine.  
Is this possible to write in the documentation?    https://docs.imgproxy.net/    
This can help some people who can't debug and don't understand why. for example: me...

**Comment by DarthSim** [comment]
It's probably better to remove these flags from the source and set them at build time. I'll think about this.
